en/starter/generator/command_line_tools_for_linux.md

```diff
@@ -0,0 +1,235 @@
+
+The tool then invokes the [Property generator](#property-generator) and prompts you to enter model properties.
+
+**Output**
+
+The tool creates a new file that defines the model, /models/model-name.json; for details, see [Model definition JSON file](model_definition_json_file.html). It also modifies the [*OpenAPI (Swagger 2.0) specification*](http://swagger.io/specification/) YAML file that describes and documents the REST APIs. By convention this file is located in the project’s definitions directory; for example, `definitions/appname.yaml`.
+
+---
+
+## Property generator
+
+**Important:** Before running this generator, you must create a CRUD type application by using the [Application generator](#application-generator). Then you must create a model using the [Model generator](#model-generator).
+
+**Synopsis**
+
+Adds a new property to an existing Swift Server Generator model.
+
+    yo swiftserver:property [options]
+
+**Options**
+
+`-h, --help`
+
+Prints the generator’s options and usage.
+
+**Interactive prompts**
+
+The tool will prompt you for the following information:
+
+-   The model to which to add the new property. If there is more than one model defined in the application you will be asked to choose from a list of the available models.
+
+-   The name of the property to add.
+
+-   The data type of the property.
+
+-   Whether the property is required; the default is no.
+
+-   Whether the property has a default value; the default is no.
+
+**Output**
+
+The tool modifies the [Model definition JSON file](model_definition_json_file.html) to add the specified property to the specified model. It also modifies the [*OpenAPI (Swagger 2.0) specification*](http://swagger.io/specification/) YAML file that describes and documents the REST APIs; for example, `definitions/appname.yaml`.
+
+---
+
+## Refresh generator
+
+**Important:** Before running this generator, you must create a Swift Server Generator application by using the [Application generator](#application-generator).
+
+**Synopsis**
+
+Regenerates your Swift Server Generator application.
+
+    yo swiftserver:refresh
+
+**Options**
+
+`-h, --help`
+
+Prints the generator’s options and usage.
+
+**Output**
+
+The tool regenerates your Swift Server Generator application; the files which are updated vary according to the capabilities of your application.  The following application files will not be updated:
+
+-		config.json
+-		main.swift
+-		Package.swift
+
+---
+
+## Build generator
+
+**Important:** Before running this generator, you must create a Swift Server Generator application by using the [Application generator](#application-generator).
+
+> ![info] In a MacOS environment the build generator will pass additional linker options to the compiler: **-Xlinker -lc++**. for this reason, running **yo swiftserver:build** is preferred over the native **swift build** command.
```

Comment:
This build option is no longer required. Could mention in a troubleshooting section maybe, but I think it's only for some specific versions of Swift pre-3.1.1.

Reply:
done